airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/GET_FLAT_FILE_ALL_ORDERS_DATA_BY_LAST_UPDATE_GENERAL.json

```diff
@@ -0,0 +1,103 @@
+{
+  "title": "Flat File All Orders Data Reports (by last update)",
+  "description": "Flat File All Orders Data by Last Update Date General Reports",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
```

Comment:
Please make sure all the fields your defined are really nullable according to the API reference. I'm not sure that the order id can be nullable.

Reply:
The API reference ([schema](https://sellercentral.amazon.com/help/hub/reference/201648780) and [reports](https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reporttype-values.md#order-tracking-reports)) are not fully documented nor updated :disappointed:. In fact, it doesn't provide the type and nullability of the fields. 

So, I think the best approach is to let all fields nullable. Maybe we can assume that order-id (which for the real report is `amazon-order-id`) and the cursor field `last-updated-date` can't be nullable.

Make sense to you?